File: c_cpp/kin/0001/0001.c
/*
https://kin.naver.com/qna/detail.nhn?d1id=1&dirId=104&docId=329415766&mode=answer
*/
#include <stdio.h>

int main() {
	int max = 0, min = 100;
	int i, n, num;
	printf("학생 수를 입력하세요 : ");
	scanf("%d", &n);
	printf("점수를 입력하세요(0~100) : ");
	for (i = 0; i < n; i++) {
		scanf("%d", &num);
		if (num > max)
			max = num;
		if (num < min)
			min = num;
	}
	printf("1등은 %d점, 꼴찌는 %d점\n", max, min);
	return 0;
}

File: python/0001.py
'''
https://kin.naver.com/qna/detail.nhn?d1id=1&dirId=10402&docId=329335646&page=1#answer1
누진세 계산기
~200kwh : 100/kwh
201~300kwh : 200/kwh
200kwh~ : 300/kwh

'''

def electricity(kwh):
    pay = 0
    if kwh<=200:
        pay = 100 * kwh 
    elif 200 < kwh <= 300:
        pay = 100*200 + 200*(kwh%200)
    else:
        pay = 100*200 + 200*100 + 300*(kwh%300)
    return pay


kwh = int(input("사용량: "))
print("요금 =",electricity(kwh))
